src/components/StudyPlanCard.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import { useNavigate } from "react-router-dom";
```

Comment:
This is needed to redirect on row 30

Reply:
Hey, thank you for revewing it. I just added that line, I deleted by mistake

Reply:
Your welcome